src/box/index.h

```diff
@@ -252,6 +252,14 @@ box_tuple_extract_key(box_tuple_t *tuple, uint32_t space_id,
 
 /** \endcond public */
 
+/**
+ * Get position of tuple in index (multikey and func indexes are not supported).
```

Comment:
Only arguments that supposed to start iteration after position are called `after` (excluding some helpers like `normalize_after`). I don't find it confusing, but maybe I'm just used to it. Do you want to rename all the arguments in C, including argument of `create_iterator`?

Reply:
Maybe, use `pos` for the argument passed and returned by iterator methods (`create_iterator`, `iterator_position`) and use `packed_pos` for the argument passed to `box_select` (i.e. for a packed position object)? Then there would be no confusion. Anyway, better discuss this with @alyapunov.

Reply:
Discussed with @alyapunov. Agreed that we want to use the same names for variables denoting the same kind of thing **everywhere** to avoid confusion:
- Position key without headers: `pos, pos_end`.
- Packed position key: `packed_pos, packed_pos_end`.

`after` shouldn't be used at all.

Reply:
Thank you! I removed all `after` arguments and variables in C, and renamed both helpers `normalize_after` to `normalize_position`.